### PR TITLE
DOC: release

### PR DESCRIPTION
**Pull request by ke-zhang-rd**
Summary all commit.

### CODE REVIEW

File: doc/source/release-notes.rst [hunk 1]
```diff
@@ -2,6 +2,19 @@
 Release History
 ***************
 
+v1.0.0 (2019-06-03)
```
Comment (tacaswell) [inline doc/source/release-notes.rst]: should be 0.4.0

Comment (danielballan) [reply doc/source/release-notes.rst]: @ke-zhang-rd I think you marked this as resolved but you did not comment. Per today's discussion, I think we now agree that this should be 0.4.0.

File: doc/source/release-notes.rst [hunk 2]
```diff
@@ -2,6 +2,19 @@
...
+===================
+
+Features
+--------
+* Rewrite approach to logging.
+* Add precision to motor_ph in mini_beamline.
```
Comment (danielballan) [inline doc/source/release-notes.rst]:
```suggestion
* Add precision to motor_ph in mini_beamline example IOC.
```

File: doc/source/release-notes.rst [hunk 3]
```diff
@@ -2,6 +2,19 @@
...
+
+Fixes
+-----
+* Scan field write in server.
```
Comment (danielballan) [inline doc/source/release-notes.rst]: Can you expand on this? I actually can't recall what this was about.

File: doc/source/release-notes.rst (hunk 1 above)
Comment (ke-zhang-rd) [reply doc/source/release-notes.rst]: Yes, I updated on cantrips but forgot here, good catch

File: doc/source/release-notes.rst (hunk 3 above)
Comment (ke-zhang-rd) [reply doc/source/release-notes.rst]: @danielballan 
https://github.com/caproto/caproto/commit/314617bd809c1d0ff883814832a39e8709d92e7e
Do you prefer more details in docs?

Comment (danielballan) [reply doc/source/release-notes.rst]:
```suggestion
* Fix bug in `scan_rate` that raised errors when it was written to
```

File: doc/source/release-notes.rst [hunk 4]
```diff
@@ -2,6 +2,19 @@
 Release History
 ***************
 
+v0.4.0 (2019-06-03)
```
Comment (danielballan) [inline doc/source/release-notes.rst]: Just a reminder to update the date when you tag

File: doc/source/release-notes.rst [hunk 5]
```diff
@@ -2,6 +2,19 @@
...
+===================
+
+Features
+--------
+* Rewrite approach to logging.
```
Comment (danielballan) [inline doc/source/release-notes.rst]:
```suggestion
* Rewrite approach to logging. See :doc:`loggers` for details.
```

File: doc/source/release-notes.rst [hunk 6]
```diff
@@ -2,6 +2,19 @@
...
+* Add precision to motor_ph in mini_beamline example IOC.
+
+Bug Fixes
+---------
+* Scan field write in server.
+* Handle case where filter is None or empty.
```
Comment (danielballan) [inline doc/source/release-notes.rst]:
```suggestion
* Respond correctly when channel filter is set but empty.
```

**Comment by tacaswell** [comment]
The infinitely queued extra job looks like github and azure webhooks (or what ever they new apps are using to communicate) got out of sync with each other.

**Pull request merged**